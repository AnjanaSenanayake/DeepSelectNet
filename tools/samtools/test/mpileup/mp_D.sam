@SQ	SN:z	LN:13
@CO	TAGCTTAGCAGGT ref
@CO	 AGCTTAGCAGG
@CO	 AGCT***CAGG
@CO	 **CTTAGCA**
@CO	 01234567890  qual
@CO	
@CO	Note ending on D gives reported quality 0 ('!')
s1	0	z	2	0	11M	*	0	0	AGCTTAGCAGG	01234567890
s2	0	z	2	0	4M3D4M	*	0	0	AGCTCAGG	01237890
s3	0	z	2	0	2D7M2D	*	0	0	CTTAGCA	2345678
