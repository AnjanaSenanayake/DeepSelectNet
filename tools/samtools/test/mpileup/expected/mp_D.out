z	2	A	3	^!.^!.^!*	002
z	3	G	3	..*	112
z	4	C	3	...	222
z	5	T	3	..-3TAG.	333
z	6	T	3	.*.	474
z	7	A	3	.*.	575
z	8	G	3	.*.	676
z	9	C	3	...	777
z	10	A	3	...-2GG	888
z	11	G	3	..*	99!
z	12	G	3	.$.$*$	00!
